docs/providers/aws/events/apigateway.md

```diff
@@ -191,6 +191,41 @@ functions:
             identityValidationExpression: someRegex
 ```
 
+You can also configure an existing Cognito User Pool as the authorizer, as shown
+in the following example:
+
+```yml
+functions:
+  create:
+    handler: posts.create
+    events:
+      - http:
+          path: posts/create
+          method: post
+          authorizer:
+            arn: arn:aws:cognito-idp:us-east-1:xxx:userpool/us-east-1_ZZZ
```

Comment:
So using `AWS_IAM` instead of `COGNITO_USER_POOL` here: https://github.com/serverless/serverless/pull/2141/files#diff-1d0146458dadc7c8c6678ea552f1f825R26 ?

The ARN test would be: `/^arn:aws:cognito-identity/`